pokemonred_puffer/global_map.py

```diff
@@ -20,3 +20,6 @@ def local_to_global(r: int, c: int, map_n: int):
     except KeyError:
         print(f"Map id {map_n} not found in map_data.json.")
         return r + 0, c + 0
+    except IndexError:
+        print(f"Coord {map_x}, {map_y} out of bounds for map id {map_n}.")
+        return -1, -1
```

Comment:
Wont this index to the lower right corner of the map? Why not use 0, 0?

Reply:
used 0, 0